src/app/components/client/ExposureCard.module.scss

```diff
@@ -84,25 +99,46 @@
         list-style: none;
         margin-left: 0;
         padding-left: 0;
-        display: flex;
-        align-items: center;
-        gap: $layout-md;
-
-        .detailsFoundItem {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-gap: $spacing-sm;
+        
+        @media screen and (min-width: $screen-md) {
           display: flex;
+          flex-direction: row;
+          align-items: center;
+        }
 
-          svg {
-            color: $color-purple-70;
+        li + li {
+         padding-left: 23px; // offset the exposure icon
```

Comment:
With magic numbers I'd like to know how the number was calculated. For example, is the exposure icon 46px and does it need to be shifted halfway to the left?

That makes it easier for reviewers and future readers to know why it is there, and whether it still needs to be there. For example, is there any reason this could not be solved with a `gap`?